core/includes/bootstrap.classes.inc

```diff
@@ -14,7 +14,7 @@ function backdrop_class_list() {
     'BatchQueue' => $path . 'batch.queue.inc',
     'BatchMemoryQueue' => $path . 'batch.queue.inc',
     'BackdropCacheArray' => $path . 'bootstrap.inc',
-    'SchemaCache' => $path . 'boostrap.inc',
+    'SchemaCache' => $path . 'bootstrap.inc',
```

Comment:
Good catch! This surely is probably causing issues. 👍🏻

Reply:
I know, right!